app/helpers/spaces_helper.rb

```diff
@@ -13,4 +13,50 @@ def inline_editable(field, title = { h2: nil }, &block)
       block: block
     }
   end
+
+  # Generates a static image of a map with a pin, based on
+  # the location of a given Space,
+  #
+  # You can set one or all of :zoom, :height or :width to
+  # change properties of the generated image
+  #
+  # For example like this:
+  # static_map_of @space, zoom: 9, height: 800
+  #
+  # Can also be combined with HTML attributes for the iamge tag
+  # static_map_of @space, zoom: 4, class: "p-4"
+  def static_map_of(space, options = {})
+    zoom = options[:zoom] || 12
+    height = options[:height] || 250
+    width = options[:width] || 400
+    html_options = {
+      **options,
+      zoom: nil
+    }
```

Comment:
I think this would be better

```
def static_map_of(space, zoom: 12, width: 400, height: 250, html_options: {}) 
....
end

# Calling could then be
static_map_of(space, zoom: 7)
static_map_of(space, width: 500, html_options: { some_option: 20 })
etc..
```

Reply:
TOTALLY agree! Thank you!

Changed it now to: 

´´´ruby
  def static_map_of(space, zoom: 12, height: 250, width: 400, **html_options)
 # foo 
end 

´´´

Reply:
Also commented the mapbox incantantations for posterity, so they are more readable for future coders (including me).

So now it reads: 

```ruby


  # Generates a static image of a map with a pin, based on
  # the location of a given Space,
  #
  # You can set one or all of :zoom, :height or :width to
  # change properties of the generated image
  #
  # For example like this:
  # static_map_of @space, zoom: 9, height: 800
  #
  # Can also be combined with HTML attributes for the iamge tag
  # static_map_of @space, zoom: 4, class: "p-4"
  def static_map_of(space, zoom: 12, height: 250, width: 400, **html_options)
    color_of_pin = 'db2777' # Tailwind pink-600
    static_map_image_url = [
      'https://api.mapbox.com/styles/v1/mapbox/streets-v11/static',
      "/pin-l-circle+#{color_of_pin}(#{space.lng},#{space.lat})", # Type, color, and position of pin
      "/#{space.lng},#{space.lat},#{zoom}", # Position of map
      "/#{width}x#{height}", # Size of map
      '?logo=false', # Hide mapbox logo
      '&@2x', # Render at 2x for retina
      "&access_token=#{ENV['MAPBOX_API_KEY']}"
    ].join
    image_tag static_map_image_url, html_options
  end

```

Reply:
Seems much better now